client/src/components/profile/Profile.tsx

```diff
@@ -57,32 +94,42 @@ export const Profile = () => {
         sx={(theme) => ({
           borderRight: 1, borderColor: 'divider',
           color: theme.palette.secondary.main,
-          height: '100%', Width: '15rem'
+          height: '100%', minWidth: '13rem'
         })}
       >
-        <Tab sx={(theme) => ({ p: 3, color: theme.palette.secondary.main })} label="My Tickets" {...a11yProps(0)} />
-        <Tab sx={(theme) => ({ p: 3, color: theme.palette.secondary.main })} label="My Events" {...a11yProps(1)} />
-        <Tab sx={(theme) => ({ p: 3, color: theme.palette.secondary.main })} label="Account Details" {...a11yProps(2)} />
+        <Tab sx={(theme) => ({ padding: '2rem 0rem', color: theme.palette.secondary.main })} label="My Tickets" {...a11yProps(0)} />
+        <Tab sx={(theme) => ({ padding: '2rem 0rem', color: theme.palette.secondary.main })} label="My Events" {...a11yProps(1)} />
+        <Tab sx={(theme) => ({ padding: '2rem 0rem', color: theme.palette.secondary.main })} label="Account Details" {...a11yProps(2)} />
       </Tabs>
       <TabPanel value={currTab} index={0}>
         {/* get events by user tickets */}
-        <Grid container spacing={3}>
-          {[0, 1, 2, 3, 4, 5, 6, 7, 8, 9].map((i) => (
-            <Grid key={i} item sm={4} md={3}>
-              <TicketCard />
-            </Grid>
-          ))}
-        </Grid>
+        {fetchingEvents ?
+          <CircularProgress />
+          :
+          <Grid container spacing={3}>
+            {/* {myTicketsEvents && myTicketsEvents?.ticket.map((i) => (
```

Comment:
delete commented code

Reply:
the commented code if for the db
there are no events right now so i commented it out and put mock data to test